### ISSUE TITLE
[Blog] Rethink blog creation process

### ISSUE THREAD

**Issue by lenybernard** [Victoire/victoire#520]
## Problem
To create a blog is still pain in the ass 😲  and it should be a way simpler :

We have almost all in the same place:
![screen shot 2016-07-13 at 10 33 34](https://cloud.githubusercontent.com/assets/618536/16796901/4a7427cc-48e5-11e6-8968-8cce9cc52eb4.png)

This is pretty nice when already setuped but not in the first times, when we have to create the infrastructure. Actually, now, when we create a blog, all seems to be perfect, the first modal is opening with the i18n fields that we fill and submit. We're redirected to the blog index which has henceforth our blog.

But when we try to create a brand new article, nothing happends and we need to open the network taband catch the error (**No result was found for query although at least one row was expected.**) :

![screen shot 2016-07-13 at 10 38 15](https://cloud.githubusercontent.com/assets/618536/16797033/ed1db146-48e5-11e6-9f9e-3cd04937177e.png)

To fix this issue now, we need to create a BusinessTemplate in the Article Tab in BusinessTemplate menu:
![screen shot 2016-07-13 at 10 40 31](https://cloud.githubusercontent.com/assets/618536/16797105/3efaaf6e-48e6-11e6-8d22-7dbb5a7406e2.png)

After that, this still does not work (and it's neither not documented anywhere) because we have to fill in database (like fat and disgusting guys we are) the parent_id column with the blog id...
![giphy 3](https://cloud.githubusercontent.com/assets/618536/16797187/a1fba816-48e6-11e6-9bea-4ddb5a04af58.gif)

### What do we need to do ?

Do we just have to add a select the blog in the `ArticleTemplateType`?
I don't think so. To my mind, an article template should be **automagically** created when we create a blog and we could find it in a tab, near the **Posts list**, **Categories list**, **Settings**.

Do you agree @clekos ?

**Comment by clekos** [comment]
Yes, I think we should create an article template automatically when we create a Blog. I think the best way is to make a new tab called "templates" where you can find it.
